node/src/components/consensus.rs

```diff
@@ -129,6 +137,9 @@ impl Display for ConsensusMessage {
                 "request for evidence of fault by {} in {} or earlier",
                 pub_key, era_id,
             ),
+            ConsensusMessage::LatestStateRequest { era_id } => {
+                write!(f, "request for the latest state in era {}", era_id)
```

Comment:
Maybe also avoid printing `EraId` here:
```suggestion
                write!(f, "request for the latest state in era {}", era_id.0)
```

Reply:
Is it printed by `Display`, too?

Reply:
I thought it was… not sure, maybe it isn't.

Reply:
Or does it print "era 42"?